lib2/get-config.ts

```diff
@@ -0,0 +1,148 @@
+import dedent from 'dedent';
```

Comment:
This is a really interseting script I stole from `wagmi`: https://github.com/wagmi-dev/wagmi/blob/main/scripts/tsup.ts.

It's not actually necessary with these changes per se, but It really helps with keeping a `package.json` up to date with changes to your modules (and it helped facilitate my desire to have a deep import of `lib2/component`.

In general this script is pretty opinionated and I certainly wouldn't use it as-is. If you want something like this I can help you edit it down to what's useful for you.